code/modules/mob/living/living.dm

```diff
@@ -552,7 +552,7 @@
 	if(restrained() || HAS_TRAIT(src, TRAIT_CANNOT_PULL))
 		stop_pulling()
 
-	var/turf/old_loc = loc
+	var/turf/old_loc = get_turf(loc) // this had better be a turf
```

Comment:
what if they arent on a turf though? like what if they move from inside a locker?

Reply:
Yeah, it's a little weird. This is kind of why I want to see it TMed if possible, since I think it should be fine, but this feels like it could touch something unexpected. Given that the variable was originally scoped as `var/turf` (and the proc args expect a turf) I feel like this might have been the intended behavior, though.

Like if they're inside a locker and move out (unless forcemoved) you probably wouldn't want the person they grabbed to be moved into the contents of the open locker, right? I'm struggling to think of an instance where moving from something's .contents should necessitate plopping someone you're grabbing back inside afterwards.